jekyll/_cci2/server/v4.4/installation/phase-2-core-services.adoc

```diff
@@ -890,6 +890,13 @@ $ base64 -w0 ~/oidc-service-jwk.json
 $ base64 -b0 ~/oidc-service-jwk.json
 ----
 +
+If you encounter an `invalid argument ~/oidc-service-jwk.json` error on macOS, use this command instead:
++
+[source,shell]
+----
+$ base64 -b0 -i ~/oidc-service-jwk.json
+----
++
```

Comment:
This should just replace the macOS instruction altogether.

Reply:
(for context, I use Linux for development, not macOS, so I had just looked up the manpages for BSD `base64` when I wrote this docs and evidently made an error, let's just replace the invalid command)

Reply:
@christian-stephen Thanks for taking a look at this! Just replaced the command to include `-i` option for macOS command. 🙇